### ISSUE TITLE
Issue with React v15+

### ISSUE THREAD

**Issue by mikesea** [Shopify/sprockets-commoner#19]
Tried setting this up on a fresh Rails app using React v15.0.2 and I'm getting the following error in the browser console:

```
Uncaught ReferenceError: _commoner_module__node_modules$fbjs$lib$emptyFunction_js is not defined
```

I switched to React v0.14.8 and everything worked.

**Comment by bouk** [comment]
I'll take a look at this, thanks

**Comment by bouk** [comment]
I'm having no issues getting this working, are you sure everything is installed properly?

**Comment by bouk** [comment]
ah I just reproduced it nvm

**Comment by bouk** [comment]
Fixed in `0.2.6`, you might have to `rm -rf tmp/cache` in your app to make it work. Let me know if it still fails
